.github/workflows/ccip-integration-test.yml

```
@@ -32,10 +32,20 @@ jobs:
           cd ccip
           git fetch
           git checkout ccip-develop
+      - name: Get the correct commit SHA via GitHub API
+        id: get_sha
+        run: |
+          if [ "${{ github.event_name }}" == "pull_request" ]; then
+            COMMIT_SHA=${{ github.event.pull_request.head.sha }}
+          else
+            COMMIT_SHA=$(curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+              "https://api.github.com/repos/${{ github.repository }}/commits/${{ github.ref }}" | jq -r .sha)
```

Does providing the branch name work?
```suggestion
            COMMIT_SHA=ccip-develop
```

---

I guess this works so long as we only ever run this workflow on merges to ccip-develop and pull requests, and not just branch pushes. If we wanna keep it like that its fine with me 😄

---

Ah actually this might have to be `latest` but will test

---

Good point, maybe something like this?
```
COMMIT_SHA=${{github.GITHUB_REF_NAME}}
```

---

ref name might work for PRs and Merges

---

Tried it out here and doesn't work unfortunately: https://github.com/smartcontractkit/chainlink-ccip/actions/runs/10457686236/job/28957870251